src/ixdat/quant_patch.py

```diff
@@ -0,0 +1,99 @@
+"""Patches corresponding to code that should be added to Spectro Inlets quantification"""
+
+
+def append_sensitivity_factors(*args):
+    """Recursively append elements in a list"""
+    sf = append_two_sensitivity_factors(args[0], args[1])
+    if len(args) == 2:
+        return sf
+    remaining_args = [sf] + list(args[2:])
+    return append_sensitivity_factors(*remaining_args)
+
+
+def append_two_sensitivity_factors(sf_1, sf_2):
+    """This should be incorporated into __add__ of SensitivityFactor and SensitivityList
+
+    Appends all the sensitivity factors in sf_1 and sf_2, which can each be a single
+    SensitivityFactor (or CalPoint) or a SensitivityList (or Calibration).
+
+    Args:
+        sf_1 (SensitivityFactor or SensitivityList): The first sensitivity factor or list
+            thereof.
+        sf_2 (CalPoint or Calibration): The second sensitivity factor or list thereof
+    """
```

Comment:
Returns SensitivityList: a `SensitivityList` containing all of the sensitivity factors
        in `sf_1` and `sf_2`. This will be a `Calibration` unless both `sf_1` and `sf_2`
        are both themselves `SensitivityList`s